README.md

```diff
@@ -1,165 +1,88 @@
-# Recommendation ITU-R P.2108-1 - U.S. Reference Implementation #
+# Recommendation ITU-R P.2108 - U.S. Reference Implementation #
 
-Persistent Identifier: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7114033.svg)](https://doi.org/10.5281/zenodo.7114033)
+<!-- TODO on v1.0 release created: add updated DOI badge -->
+[![GitHub Actions Unit Tests Status][gh-actions-test-badge]][gh-actions-test-link]
+[![C++ API Reference][gh-actions-docs-badge]][gh-actions-docs-link]
```

Comment:
Broken image.
![image](https://github.com/user-attachments/assets/02f244d2-c6c7-4197-a3d4-a68c90192145)

Reply:
This link will automatically work once this PR is merged. The linked image is generated by the GitHub action which generates the Doxygen documentation. Since that action has not run in this repository yet (since it has not been merged), this image link is currently broken. When the action runs and succeeds, the generated badge will appear and look like [this (example in private dev repo)](https://github.com/NTIA/p2108-extra/actions/workflows/doxygen.yml/badge.svg). Similarly, the link in the README to the Doxygen site is broken until this branch is merged and the GitHub Pages site is published.

Reply:
This is now adjusted so that the README badges always display the status badges for workflow runs _on the main branch_. This way, we won't show "test failing" when the main branch/current release has tests passing but ongoing dev work has a recent test failing. This also makes it look nicer when there are _no_ workflow runs for the main branch (i.e., right now before this PR is merged):

![image](https://github.com/user-attachments/assets/bf4be996-85a9-48d1-845b-afc034773a90)